### PR TITLE
samples: create Pub/Sub BigQuery subscription

### PR DESCRIPTION
**Pull request by kamalaboulhosn**

### CODE REVIEW

File: pubsub/api/Pubsub.Samples.Tests/CreateBigQuerySubscriptionTest.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+// limitations under the License.
+
+using Xunit;
+
+[Collection(nameof(PubsubFixture))]
+public class CreateBigQuerySubscriptionTest
+{
+    private readonly PubsubFixture _pubsubFixture;
+    private readonly CreateBigQuerySubscriptionSample _createBigQuerySubscriptionSample;
+    private ListSubscriptionsSample _listSubscriptionsSample;
```
Comment (amanda-tarafa) [inline pubsub/api/Pubsub.Samples.Tests/CreateBigQuerySubscriptionTest.cs:22]: nit: I think you are not using this one. The assert you have is fine, no need to actually list all the subscriptions. Just remove this one.

**Review by amanda-tarafa — APPROVED** [review]
LGTM, except for a nit and the fact that your commit history is wrong. I think you need to rebase your branch.

File: pubsub/api/Pubsub.Samples.Tests/CreateBigQuerySubscriptionTest.cs (hunk 1 above)
Comment (kamalaboulhosn) [reply pubsub/api/Pubsub.Samples.Tests/CreateBigQuerySubscriptionTest.cs]: Done.

**Comment by kamalaboulhosn** [comment]
Rebased, though I'm not sure it changed anything.

**Comment by amanda-tarafa** [comment]
Is the branch you are rebasing on up to date?

**Comment by kamalaboulhosn** [comment]
@amanda-tarafa Yes, it is up to date. Can you tell me more specifically what the issue is? The "Files changed" looks correct. So when it gets squashed and merged, I expect everything will be fine.

**Comment by amanda-tarafa** [comment]
We usually do `rebase and merge` to respect the commits of the PR if those are meaningful. But is true that in this case we would have a single commit anyway so I can squash and merge. Doing so now...

**Pull request merged**